docs/syscall_descriptions_syntax.md

```diff
@@ -188,6 +187,8 @@ type bool32	int32[0:1]
 type bool64	int64[0:1]
 type boolptr	intptr[0:1]
 
+type fileoff[BASE] BASE
```

Comment:
Nice! Will be useful to do other similar typedef's.

Reply:
Any ideas in mind?